app/views/event/register/register.html.haml

```diff
@@ -0,0 +1,29 @@
+-#  Copyright (c) 2023, CEVI Schweiz, Pfadibewegung Schweiz,
+-#  Jungwacht Blauring Schweiz, Pro Natura, Stiftung für junge Auslandschweizer.
+-#  This file is part of hitobito_youth and
+-#  licensed under the Affero General Public License version 3
+-#  or later. See the COPYING file at the top-level directory or at
+-#  https://github.com/hitobito/hitobito_youth.
+
+- title t(".title.#{manager ? 'manager' : 'participant'}")
+
+- unless manager
+  = render 'event/participations/step_wizard', step: 1
+
+- return_url = manager ? group_event_path(group, event) : new_group_event_participation_path(group, event)
```

Comment:
Das war eigentlich bewusst anders spezifiziert. Ich gehe davon aus, dass die Leute manchmal den falschen Button erwischen weil sie nicht  fertig lesen (oder einfach ENTER drücken). Das Formular sollte sich in beiden Fällen exakt gleich verhalten, bis auf die Kosmetik mit dem Titel, Flash Message und den Labels "Vorname" vs. "Vorname der erziehungsberechtigten Person" etc.. Also sobald der Elternzugang aktiviert ist sollte man in keinem Fall mehr direkt auf den `new_group_event_participation_path` kommen, sondern immer nur zur Anzeige des Events. Dort muss man nochmals ganz bewusst im "Anmelden"-Dropdown auf seinen eigenen Namen klicken, oder eben auf "Neues Kind anmelden".

Von mir aus könnte man alle Bedingungen (ausser den Labels) die du auf `manager` programmiert hast stattdessen aufs Feature Gate achten lassen. Also auch der Step Wizard sollte nie auftauchen solange das FeatureGate aktiviert ist.